README.md

```diff
@@ -1,14 +1,18 @@
-# Recursion Context
+# sebastian/recursion-context
```

Comment:
Since touching this file, I took the freedom to streamline the content a bit.

Let me know if you would prefer to revert these changes.